### ISSUE TITLE
Doubt: Want to know How I can use Protege and Jason together?

### ISSUE THREAD

**Issue by Aviral190694** [jason-lang/jason#16]
I know its a broad question but I am not able to get the right answer to it. I used Jacomo too but still not able to do it. I want to use the ontology I created using Protege with Jason so that Agents can get semantic information.

**Comment by rbordini** [comment]
Hi Aviral,

I'm CCing my student Alison who might have the code for an artifact that 
would allow you to access an ontology in a JaCaMo system. Alison, if you 
don't have the code, do you know if it's stable enough and who in the 
group would have it?

Cheers,

Rafael

**Comment by Aviral190694** [comment]
Rafeal

That would be a great help. I am kind of stuck here. A sample will do great help. I am looking forward to see Alison's reply.

Thanks a ton.

**Comment by rbordini** [comment]
Hi Aviral,

The SeaTeaMS group have developed this code as part of a project
(2013-2015) sponsored by a private company and, unfortunately, the
implementation is intellectual property of this company, i.e., we are not
allowed to share it.
However, we have published this paper -- http://www.meneguzzi.eu/felipe
/pubs/iat-ontology-artifact-2015.pdf -- which describes our solution, and
it could be helpful if you intend to implement a similar approach.
I don't know the details of this implementation, given that my role was
help to define the operations we needed to implement the decision-making
for that application. However, the main role of that code was to
encapsulate the OWL API in an CArtAgO Artifact, as described in the paper,
thus I think that, depending on your need, OWI API could be very useful.

Best regards,
Alison

2017-10-25 1:14 GMT+01:00 Rafael H Bordini <rafael.h.bordini@gmail.com>:
